.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,36 +1,65 @@
 <!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
 
 ## About the PR
-<!-- What did you change? -->
+<!-- Summarize what you changed here. -->
 
-## Why / Balance
-<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
+## Reason for change
+<!-- Explain why it was changed. Link any relevant discussions or issues. -->
+
+## Effect on game balance
```

Comment:
I've broken the reason for the change and the effect on balance into two sections. This makes PRs easier to read and encourages folks to specifically explain the balance change away from "balance is bad" which the old combined section encouraged.

Reply:
Or rather:

"Nukie zombie ops are not fun." is a reason TO change something but it's not the impact OF changing something.